bin/code_samtools_flagstat.sh

```
@@ -0,0 +1,32 @@
+input_bam_index="$2"
+pathToBin="nextflow-bin"
+input_bam_prefix=${input_bam%%.*}
+
+
+
+echo "Value of input_bam: '$input_bam'"
+echo "Value of input_bam_index: '$input_bam_index'"
+
+echo "installing packages"
+
+export SAMTOOLS=${pathToBin}/samtools-1.16*
```

Shellcheck (0.10.0)

[warning] Brace expansions and globs are literal in assignments. Quote it or use an array.

(SC2125)